src/model/navigation.model.ts

```diff
@@ -1,7 +1,7 @@
 export interface Navigation {
   [x: string]: {
     currentPage: string;
-    previousPage: string;
-    nextPage: string;
+    previousPage?: string | string[];
```

Comment:
Wondering about this change - is this now supporting multiple previous pages?

Reply:
Navigation will not be used for this story, it will be reverted